package.json

```diff
@@ -146,6 +152,12 @@
         "key": "ctrl+enter",
         "mac": "cmd+enter",
         "when": "editorTextFocus && editorLangId == markdown"
+      },
+      {
+        "command": "memo.openReferenceInNextColumn",
+        "key": "ctrl+shift+enter",
+        "mac": "cmd+shift+enter",
+        "when": "editorTextFocus && editorLangId == markdown"
```

Comment:
I suggest adjusting `when` condition to ensure reference context. This shortcut might collide with other shortcuts, at least in my `keybidings.json`. There is already [an example](https://github.com/svsool/vscode-memo/blob/1ab1759e9ebc5e2d6b39273ca53c30f67ae450ac/src/features/referenceContextWatcher.ts) of custom context that Memo implements called `memo:refUnderCursorExists`, but it's true only when a certain link (file) exists on disk, so I think we need something like `memo:ref<Focused|Hovered>`.

![image](https://user-images.githubusercontent.com/1452141/104364526-644c4800-551f-11eb-8925-d1db1b391bf7.png)

Reply:
makes great sense. thanks! will add this.

Reply:
added this in new commits. also added context menu "Open link to the side" for link/ref.

![image](https://user-images.githubusercontent.com/21694444/104861330-40548200-58e4-11eb-9d46-9f766238463c.png)

Reply:
awesome, and thanks for extending context menu!